### DISCUSSION TITLE
Route to user info without id

### DISCUSSION THREAD

**Discussion by jaribu** [hagopj13/node-express-boilerplate#105] [Q&A]
Hello,
I try to connect your beautifull boilerplate with nuxtjs/auth.
How can I make this route for a user who have already make a login or register with a token ?
Like /user/me

**Comment by hagopj13** [comment]
@jaribu sounds like a good idea. You can create a new endpoint (as you suggested `/users/me` which looks at the auth token to figure out who the user is.
